### PR TITLE
Add CMake compilation instructions

### PR DESCRIPTION
**Pull request by heinrich5991**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,33 +25,41 @@ To clone the libraries if you have previously cloned ddnet without them:
...
     cmake ..
     make
 
-DDNet requires additional libraries, that are bundled for the most common platforms (Windows, Mac, Linux, all x86 and x86_64). The bundled libraries are now in the ddnet-libs submodule.
+DDNet requires additional libraries, that are bundled for the most common platforms (Windows, Mac, Linux, all x86 and x86\_64). The bundled libraries are now in the ddnet-libs submodule.
 
 You can install the required libraries on your system, remove the `config.lua` and `bam` should use the system-wide libraries by default. You can install all required dependencies and bam on Debian and Ubuntu like this:
 
-    apt-get install libsdl2-dev libfreetype6-dev libcurl4-openssl-dev libogg-dev libopus-dev libopusfile-dev bam
+    apt-get install libsdl2-dev libfreetype6-dev libcurl4-openssl-dev libogg-dev libopus-dev libopusfile-dev bam cmake
```
Comment (def-) [inline README.md]: Remove bam here and later and update instructions to cmake?

File: README.md [hunk 2]
```diff
@@ -25,33 +25,41 @@ To clone the libraries if you have previously cloned ddnet without them:
...
 
+Building on Windows with Visual Studio
+--------------------------------------
+
+Download and install some version of [Microsoft Visual Studio](https://www.visualstudio.com/) (as of writing, MSVS Community 2017) with **C++ support**, install [Python 3](https://www.python.org/downloads/) **for all users** and install [CMake](https://cmake.org/download/#latest).
+
+Start CMake and select the source code folder (where DDNet resides, the directory with `CMakeLists.txt`). Additionally select a build folder, e.g. create a build subdirectory in the source code directory. Click "Configure" and select the Visual Studio generator (it should be pre-selected, so pressing "Finish" will suffice). After configuration finishes and the "Generate" reactivates, click it. When that finishes, click "Open Project". Visual Studio should open. You can compile the DDNet client by right-clicking the DDNet project (not the solution) and select "Select as StartUp project". Now you should be able to compile DDNet by clicking the green, triangular "Run" button.
```
Comment (def-) [inline README.md]: Maybe screenshots would help more

Comment (heinrich5991) [reply README.md]: To be honest, I don't really know how to make a good screenshot guide. Additionally, I don't think we should place the pictures in this repository. Maybe someone could produce a screenshot guide in the forums/the wiki and we put a link into the README?

**Comment by heinrich5991** [comment]
@def- Added Linux/macOS CMake build instructions.

**Comment by def-** [comment]
Thanks

**Pull request merged**